jit-compiler/src/codegen.rs

```diff
@@ -142,8 +142,24 @@ impl<'a, T: FieldElement> CodeGenerator<'a, T> {
             _ => return Err(format!("Expected function type, got {ty}")),
         };
 
+        let generics = if vars.is_empty() {
+            String::new()
+        } else {
+            // TODO support actual bounds
```

Comment:
planned for one of the following PRs? if not, maybe a more descriptive TODO?

Reply:
It's not really properly implemented on the PIL side yet. We will notice when it breaks, but I can extend the comment.